models/VideoModel.js

```diff
@@ -0,0 +1,11 @@
+var VideoModel = Backbone.Model.extend({
+  defaults: function () {
+    return {
+        id: '',
+        img: '',
+        title: '',
+        description: '',
+        isMainVideo: false
```

Comment:
You could update the selected video on click without using the `isMainVideo` property - this is a slightly more roundabout approach